README.md

```diff
@@ -11,7 +11,7 @@ $ cd django_zbp
 $ npm install bower
 $ pip install -r requirements.txt
 $ cp django_zbp/settings/local.sample.py django_zbp/settings/local.py
-$ edit django_zbp/settings/local.py
+$ edit django_zbp/settings/local.py -> set GDAL_LEGACY
```

Comment:
What should be set into this config option? Could you provide some more comment in the readme?